samples/msal-node-samples/standalone-samples/auth-code/readme.md

```diff
@@ -0,0 +1,203 @@
+
+Before proceeding, go to the Azure portal, and open the app registration for this app.
+
+#### **Client ID**
+Within the "Overview" you will see a GUID labeled **Application (client) ID**.  Copy this GUID to the clientId field in the config.
+
+Click the **Authentication** link in the left nav.
+
+#### **Authority**
+Check that supported account types are: **Accounts in any organizational directory (Any Azure AD directory - Multitenant) and personal Microsoft accounts (e.g. Skype, Xbox)**
+
+If so, then leave the default setting for authority as `https://login.microsoftonline.com/common`
+
+For other supported account types, review the other [Authority options](https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-client-application-configuration).  Unless there is a specific need to restrict users of your app to an organization, we strongly suggest that everyone use the default authority.  User restrictions can be placed later in the application flow if needed.
+
+#### **Client Secret**
+
+This secret helps prevent third parties from using your app registration.
+Click on `Certificates and Secrets` in the left nav.
+Click `New Client Secret` and pick an expiry.
+Click the `Copy to Clipboard` icon, and add the secret to the config object in index.js.
+
+🎉You have finished the basic configuration!🎉
+
+### Executing the application
+
+From the command line, let npm install any needed dependencies.  This only needs to be done once.
+
+```bash
+$ npm install
+```
+1. Once the dependencies are installed, you can run the sample application by using the following command:
+
+```bash
+$ npm start
+```
+
+2. Navigate to http://localhost:3000 (or whatever port number specified) with the browser of your choice.
+
+### Customizing the application
+
+To customize the start script, review the `package.json` file.
+
+## Adding this scenario to an existing application
+
+### Import the Configuration Object
+
+If you set up the sample with your app registration, you may be able to copy this object directly into your application.  
+
+
+```js
+const config = {
+    auth: {
+        clientId: "12d77c73-d09d-406a-ae0d-3d4e576f7d9b",
+        authority: "https://login.microsoftonline.com/common",
+        clientSecret: ""
+    },
+    system: {
+        loggerOptions: {
+            loggerCallback(loglevel, message, containsPii) {
+                console.log(message);
+            },
+            piiLoggingEnabled: false,
+            logLevel: msal.LogLevel.Verbose,
+        }
+    }
+};
+```
+
+### Configure Dependencies
+
+Add the dependency on MSAL Node to your Node app.
+
+```js
+const msal = require('@azure/msal-node');
+```
+
+### Initialize MSAL Node at runtime
+
+
+Initialize the app object within your web app.
+
+```js
+const pca = new msal.ConfidentialClientApplication(config);
```

Comment:
`pca` and `client secret` - is that a correct combo?

Reply:
I should change the variable name.

Reply:
its not and good catch, i called this out when i did the demo sample for support.  easy 3 spots to change pca to cca